docs/releasenotes/v0.1.0.md

```diff
@@ -10,7 +10,7 @@
 - Housekeeping (#229)
```

Comment:
I am unsure if we should alter the releasenotes, because I guess that they are generated from the commit/pull-request-titles. If so, then altering the description afterwards could cause confusion.

Maybe we should ignore this file. What do you think?

Reply:
yeah... that's the thing: I haven't found a solution for the spell-check to ignore special paths/files :-(

Reply:
Apparently, there could be a solution https://github.com/rojopolis/spellcheck-github-actions/issues/26